examples/test/model/index.html

```diff
@@ -11,13 +11,16 @@
       <a-assets>
```

Comment:
scope creep alert 😉 

Added the meta title/description, but I'm getting errors trying to do `<a-text value="foo"></a-text>":

```
THREE.WebGLShader: gl.getShaderInfoLog() fragment WARNING: 0:102: '
' : extension directive should occur before any non-preprocessor tokens 
  1: precision highp float;
2: precision highp int;
3: #define SHADER_NAME ShaderMaterial
4: #define GAMMA_FACTOR 2
5: #define NUM_CLIPPING_PLANES 0
6: #define UNION_CLIPPING_PLANES 0
7: uniform mat4 viewMatrix;
...
```